src/controllers/materi.controller.js

```
@@ -0,0 +1,108 @@
+};
+
+// Update Materi: Mengupdate materi berdasarkan id
+exports.updateMateri = async (req, res) => {
+  const { id } = req.params;
+  try {
+    // Ambil nilai baru dari req.body (termasuk option update untuk guru_id/mapel_id jika perlu)
+    const { judul, deskripsi, guru_id, mapel_id } = req.body;
+    const file_url = req.file ? req.file.path : undefined;
+
+    // Siapkan data yang akan diupdate
+    let data = { judul, deskripsi, guru_id, mapel_id };
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'data' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)